src/middlewares/auth.js

```diff
@@ -0,0 +1,29 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
```

Comment:
@gmemmy I think `dotenv.config()` should be called on the entry file `src/index.js`.
If it is called on the entry file, we wouldn't have to call it again, in many other places.

> One dotenv.config() to rule them all.

Reply:
dotenv [documentation](https://www.npmjs.com/package/dotenv#usage)

> As early as possible in your application, require and configure dotenv.